server/tables/tables.go

```diff
@@ -171,6 +171,8 @@ type Invocation struct {
 	RemoteExecutionEnabled bool
 
 	Tags string
+
+	ParentInvocationID string
```

Comment:
should we store this as a `UUID` `[]byte` instead, similar to `InvocationUUID` above? (IIRC @luluz66 has more context for these UUID fields)

Reply:
InvocationUUID was added initially for the targetStatus table in mysql. It was a foreign key for that table. We use Binary(16) to save space.

Reply:
OK, I think these are fine as-is then. Thanks for explaining!

Reply:
@luluz66 mentioned that we might want to store parent invocation ID as a UUID in clickhouse, so that its format matches InvocationUUID if we every want to join on the field or compare them

@luluz66 @bduffany Could you take another look and make sure this looks right?